drivers/crc/crc_stm32.c

```diff
@@ -0,0 +1,232 @@
+/*
+ * Copyright (c) 2024 Brill Power Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(crc_stm32, CONFIG_CRC_HW_LOG_LEVEL);
+
+#include <errno.h>
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/clock_control/stm32_clock_control.h>
+#include <zephyr/drivers/crc.h>
+#include <zephyr/sys/byteorder.h>
+
+#include <soc.h>
+#include <stm32_ll_crc.h>
+
+#define DT_DRV_COMPAT st_stm32_crc
+
+struct crc_stm32_cfg
+{
```

Comment:
`struct` braces need to follow the name.

```suggestion
struct crc_stm32_cfg {
```